Source/WebKit/WebProcess/WebPage/FindController.cpp

```diff
@@ -293,11 +289,23 @@ void FindController::findString(const String& string, OptionSet<FindOptions> opt
     }
 
     RefPtr<WebPage> protectedWebPage = m_webPage;
+
+#if ENABLE(IMAGE_ANALYSIS)
+    if (completionHandler) {
```

Comment:
Instead of basing this code off of the presence of `completionHandler`, it might be nicer to take a named `enum class` instead, like:

```
enum class TriggerImageAnalysis : bool { No, Yes };
```